pkg/serviceprovider/serviceprovider.go

```diff
@@ -136,33 +141,54 @@ func (f *Factory) FromRepoUrl(ctx context.Context, repoUrl string, namespace str
 	return hostCredentialProvider, nil
 }
 
-func (f *Factory) getBaseUrlsFromConfigs(ctx context.Context, namespace string) (map[config.ServiceProviderType][]string, error) {
-	secretList := &corev1.SecretList{}
-	listErr := f.KubernetesClient.List(ctx, secretList, client.InNamespace(namespace), client.HasLabels{
-		api.ServiceProviderTypeLabel,
-	})
-	if listErr != nil {
-		return nil, fmt.Errorf("failed to list oauth config secrets: %w", listErr)
+// KnownSaasUrls represents immutable map to translate SAAS URLs from service provider names
+// TODO: It should be replaced by a proper solution that can be shared among oauth service and operator
+func KnownSaasUrls() map[config.ServiceProviderType]string {
```

Comment:
This should be removed after merge of https://github.com/redhat-appstudio/service-provider-integration-operator/pull/392